lib/dialects/mssql/query.js

```diff
@@ -51,7 +51,7 @@ class Query extends AbstractQuery {
           } else {
             resolve(this.formatResults());
           }
-        }, this.options.transaction.name, this.options.isolationLevel);
+        }, this.options.transaction.name, Utils.mapIsolationLevelStringToTedious(this.options.isolationLevel));
```

Comment:
We should prefer `this.lib` over `require('tedious')` here as dialect module path can be overridden. May be pass `this.lib` to `mapIsolationLevelStringToTedious` as second argument